src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -41,3 +65,62 @@ export class SurgingTempest extends Analyser {
 		return (statusUptime / fightUptime) * 100
 	}
 }
+
+export class StormsEye extends CoreGauge {
+	static override handle = 'stormseye'
+
+	@dependency private suggestions!: Suggestions
+
+	private surgingTempest = this.add(new TimerGauge({
+		maximum: 60,
+	}))
+
+	private earlyRefreshCount = 0
+
+	override initialise() {
+		super.initialise()
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.action(this.data.matchActionId(SURGING_TEMPEST_GENERATORS)),
+			(_) => this.extendSurgingTempest(SURGING_TEMPEST_GENERATION_AMOUNT)
+		)
+
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.action(this.data.matchActionId(SURGING_TEMPEST_EXTENDERS)),
+			(_) => this.extendSurgingTempest(SURGING_TEMPEST_EXTENSION_AMOUNT, true)
+		)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private extendSurgingTempest(amount: number, onlyIfRunning ?: boolean) {
+		onlyIfRunning === undefined ? false : onlyIfRunning
+
+		if (this.surgingTempest.remaining > SURGING_TEMPEST_EARLY_REFRESH_GRACE) {
+			this.earlyRefreshCount++
+		}
+
+		if ((this.surgingTempest.expired || this.surgingTempest.paused) && !onlyIfRunning) {
+			this.surgingTempest.start()
+		}
+		this.surgingTempest.extend(amount, onlyIfRunning)
+	}
+
+	private onComplete() {
+		if (this.earlyRefreshCount > 0) {
+			this.suggestions.add(new TieredSuggestion({
+				content: <Trans id="war.stormseye.suggestions.overwrite.content">
+				Avoid refreshing <DataLink status="SURGING_TEMPEST" /> too early
+				</Trans>,
+				why: <Trans id="war.stormseye.suggestions.overwrite.why">
+					<DataLink action="STORMS_EYE" /> Does less damage and generates less Beast Gauge than <DataLink action="STORMS_PATH" />. You lost {this.earlyRefreshCount * STORMS_EYE_LOST_GAUGE} Beast Gauge over the course of the fight due to early refreshes.
```

Comment:
Why text doesn't need to explain what they should be doing, only what they did.
```suggestion
					You lost {this.earlyRefreshCount * STORMS_EYE_LOST_GAUGE} Beast Gauge over the course of the fight due to early refreshes.
```